routes/routes.go

```diff
@@ -26,5 +26,6 @@ func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, c Clients) {
 	log.Info(ctx, "adding routes")
 	r.StrictSlash(true).Path("/health").HandlerFunc(c.HealthCheckHandler)
 	r.StrictSlash(true).Path("/sixteens{uri:/.*}").Methods("GET").HandlerFunc(handlers.SixteensBulletin(*cfg, c.Render, c.Zebedee, c.ArticlesAPI))
+	r.StrictSlash(true).Path("/{uri:.*}/data").Methods("GET").HandlerFunc(handlers.BulletinData(*cfg, c.ArticlesAPI))
```

Comment:
More of a general statement/question, most of the handler functions are sent a pointer of `config` but that doesn't appear to be used. Might be worth (beginning) to strip them out if they aren't needed/used

Reply:
The config is "as received" when work resumed on this controller. I'll raise a ticket to compare its use to other controllers and see what's missing.